backend/service/k8s/event.go

```diff
@@ -34,51 +35,65 @@ func (s *svc) ListEvents(ctx context.Context, clientset, cluster, namespace, obj
 	return events, nil
 }
 
-func ProtoForEvent(cluster string, k8sEvent *corev1.Event) *k8sapiv1.Event {
-	clusterName := GetKubeClusterName(k8sEvent)
-	if clusterName == "" {
-		clusterName = cluster
```

Comment:
curious, why was this removed? (https://github.com/lyft/clutch/pull/2704/files)

Reply:
oops yeah I accidently removed that when i merged in main branch